apps/@sparrow-desktop/src/pages/workspace-page/sub-pages/SocketIoExplorerPage/SocketIoExplorerPage.ts

```diff
@@ -339,6 +340,18 @@ class SocketIoExplorerPageViewModel {
     this.compareRequestWithServer();
   };
 
+  /**
+   *
+   * @param _headers - request headers
+   */
+  public upadateEvents = async (_events: EventsValues[]) => {
+    const progressiveTab = createDeepCopy(this._tab.getValue());
+    progressiveTab.property.socketio.events = _events;
+    this.tab = progressiveTab;
+    await this.tabRepository.updateTab(progressiveTab.tabId, progressiveTab);
+    this.compareRequestWithServer();
+  };
```

Comment:
add check for events in compare function

Reply:
Resolved